### PR TITLE
[MERGE #77 FIRST] List submissions

### PR DESCRIPTION
**Pull request by helgridly**

### CODE REVIEW

File: src/main/resources/swagger/listings/submissions [hunk 1]
```diff
@@ -5,7 +5,41 @@
 	"resourcePath": "/workspaces/{workspaceNamespace}/{workspaceName}/submissions",
 	"apis": [{
 		"path": "/workspaces/{workspaceNamespace}/{workspaceName}/submissions",
-		"operations": [{
+		"operations": [
```
Comment (abaumann) [inline src/main/resources/swagger/listings/submissions]: again, thanks for updating docs

File: src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceApiServiceSpec.scala [hunk 2]
```diff
@@ -1044,6 +1044,16 @@ class WorkspaceApiServiceSpec extends FlatSpec with HttpService with ScalatestRo
       }
   }
 
+  it should "return 200 when listing submissions" in withTestDataApiServices { services =>
```
Comment (abaumann) [inline src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceApiServiceSpec.scala]: is it a 200 with empty list when there are no submissions on a workspace?

File: src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala [hunk 3]
```diff
@@ -697,6 +699,17 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, workspaceDAO:
     }
   }
 
+  def listSubmissions(workspaceName: WorkspaceName): PerRequestMessage =
+    dataSource inTransaction { txn =>
+      txn withGraph { graph =>
+        withWorkspace(workspaceName, txn) { workspace =>
+          requireAccess(GCSAccessLevel.Read, workspace, txn) {
+            RequestComplete(submissionDAO.list(workspace.namespace, workspace.name, txn).toList)
```
Comment (abaumann) [inline src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala]: so we already had the DAO for list, just no endpoint?

File: src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceApiServiceSpec.scala (hunk 2 above)
Comment (helgridly) [inline src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceApiServiceSpec.scala]: It's whatever `submissionDAO.list()` returns, so I think yes.

File: src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala (hunk 3 above)
Comment (helgridly) [inline src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala]: Yup.

**Pull request merged**